### ISSUE TITLE
When editing meta-data on a page - the Page Summary editor should be disabled if the "Auto Generate summary" flag is enabled.

### ISSUE THREAD

**Issue by natechadwick** [percussion/percussioncms#534]
Behavior:  When editing the meta-data on a page, under “Page Summary”, the Rich Text or TinyMCE editor should not be enabled and allowed to have data put in there is the flag for “Auto Generate summary” is enabled.

![page-sum](https://user-images.githubusercontent.com/1232206/181259446-3b3ae3c9-9cde-42cb-86b6-0bb323a494d4.png)

Environment

This was reproduced on the following nightly build of version 8.1.1

![page-sum-2](https://user-images.githubusercontent.com/1232206/181259489-ce66fcc1-ed5a-49ff-bd17-26c0c195105c.png)

**Comment by NxPerc2019** [comment]
When testing with the following nightly build of 8.1.1 (see screenshot), I am getting errors in the java console related to the auto summary when I view Meta-Data.

![image](https://user-images.githubusercontent.com/50147765/183920354-865fda97-4de3-4faa-9357-ecf4ba437d41.png)

![image](https://user-images.githubusercontent.com/50147765/183920625-ceacb217-1a73-4cc6-86e8-9a38e786bfab.png)

When I put the page in Edit view - I see that the Rich Text editor is not there (which is good), but then the java errors appear.
![image](https://user-images.githubusercontent.com/50147765/183920887-24e19698-6067-4e3f-97fa-b3f590b1e878.png)

When I hit save, I get the java console errors below and my server hung:
![image](https://user-images.githubusercontent.com/50147765/183921307-08afc277-041e-4280-8aba-74b106bf1638.png)

The jetty console is showing the following:
10:02:24,021 ERROR [PSSerializerUtils] Unable to load XML file.  Check for syntax problems. Error: null C:javax.xml.bind.helpers.AbstractUnmarshallerImpl:L:335 Cause:Reference is not allowed in prolog. C:com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper:L:203, Data: &lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot; standalone=&quot;yes&quot;?&gt;
&lt;RegionBranches&gt;
    &lt;regionWidgetAssociations/&gt;
    &lt;regions/&gt;
&lt;/RegionBranches&gt;

10:02:24,021 ERROR [PSAssemblerBase] Problem during assembly: Error: The validated object is null C:org.apache.commons.lang.Validate:L:222
10:03:27,196 INFO  [Aging Agent] Polling aging action at: Wed Aug 10 10:03:27 EDT 2022...

**Comment by NxPerc2019** [comment]
I retested this on the following nightly build of 8.1.1:
![image](https://user-images.githubusercontent.com/50147765/185167415-c5b7bd17-cdac-4847-a30d-1767ffe986af.png)

I am seeing a bit of a "glitch" which I also showed to Nate.  When I first open the "Edit Meta-data", I see the Rich Text editor for a moment, then disappears with the "Auto generate Summary" flag on.  Nate considers this a browser timing issue and not much we can do about it.  When I deselect the flag, the Editor appears at it should and when I enable it, the editor goes away.  It is only on initial open that it appears for a moment.  I would consider this solved.
